### PR TITLE
Allow clients to disable better chat and fix pause menu overlapping.

### PR DESCRIPTION
**Pull request by Mka0207**
Optional request to merge, let me know if you think this is good enough to be added.

### CODE REVIEW

File: lua/schat/cl_interface.lua [hunk 1]
```diff
@@ -1,3 +1,5 @@
+CreateClientConVar("disable_betterchat", "0", true, false)
```
Comment (StyledStrike) [inline lua/schat/cl_interface.lua:1]: Please rename to `customchat` to avoid confusion (`betterchat` is the name of another chat addon)

**Review by StyledStrike — CHANGES_REQUESTED** [review]
Besides the requested change, overall, this seems good.
_Thanks for contributing!_

File: lua/schat/cl_interface.lua (hunk 1 above)
Comment (Mka0207) [reply lua/schat/cl_interface.lua:1]: Sure I can do that for ya!

**Review by StyledStrike — APPROVED** [review]
Looks good

**Pull request merged**